src/chrome/content/rules/Freifunk_Gera-Greiz.xml

```diff
@@ -0,0 +1,17 @@
+<ruleset name="Freifunk Gera-Greiz">
```

Comment:
Since `Buergernetz-Gera-Greiz.de.xml` is related to this, you should add comments to document the relationship.